JAICore/jaicore-search/build.gradle

```diff
@@ -5,8 +5,7 @@ dependencies {
 	compile project(":JAICore:jaicore-graphvisualizer")
 	
 	compile group: 'org.knowm.xchart', name: 'xchart', version: '3.5.2'
-	compile group: 'it.unimi.dsi', name: 'fastutil', version: '8.2.1'
-  testCompile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
+	  testCompile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'

```

Comment:
why is this only needed for testing?

Reply:
good point, this can be removed since the library is already included via jaicore-basic.